tests/test_downsampling.py

```diff
@@ -181,3 +183,17 @@ def test_downsample_multi_channel():
     assert np.any(target_buffer != 0)
 
     assert np.all(target_buffer == joined_buffer)
+
+def test_anisotropic_mag_calculation():
+    scale_test = [(10.5, 10.5, 24), (10.5, 10.5, 21), (10.5, 24, 10.5), (24, 10.5, 10.5), (10.5, 10.5, 10.5)] * 2
+    mags_test = [Mag(1), Mag(1), Mag(1), Mag(1), Mag(1),
+                 Mag([2, 2, 1]), Mag([2, 2, 1]), Mag([2, 1, 2]), Mag([1, 2, 2]), Mag(2)]
+    mags_expected = [Mag([2, 2, 1]), Mag([2, 2, 1]), Mag([2, 1, 2]), Mag([1, 2, 2]), Mag([2, 2, 2]),
```

Comment:
The test cases would be a bit easier to read if there were structured like that:
```
[
  [input_mag1, expected_mag1],
  [input_mag2, expected_mag2],
  ...
]
```